### PR TITLE
[ADD] web_relativedelta

### PR DESCRIPTION
**Pull request by hbrunn**

### CODE REVIEW

**Comment by guewen** [comment]
:+1: (not tested)

**Comment by npiganeau** [comment]
:+1: 
Tested, works as expected, great module to get filters such as "last week" working !

**Comment by rvalyi** [comment]
sorry no review on my side, but I just wanted to mention that in v8 we now have moment.js bundled by default, so it's just something to keep in mind when migrating this to v8 later.

File: web_relativedelta/__openerp__.py [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+    be relativedelta(weeks=-1, days=1, weekday=0) etc.
+
+- all of them together in a predictable manner as detailed in
+https://labix.org/python-dateutil#head-72c4689ec5608067d118b9143cef6bdffb6dad4e
+
+Usage
+=====
+
+Simply depend on web_relativedelta and enjoy most of python's relativedelta
+functionality
```
Comment (pedrobaeza) [inline web_relativedelta/__openerp__.py:53]: Maybe you should refer that the enjoying is in the domains...

**Comment by pedrobaeza** [comment]
Are the engines the best icon for this (maybe a calendar, or two sheets of the calendar putting one against the other...).

From the functional point of view, I made some tests and works well, but forget to comment here, so LGTM (functional).

**Comment by max3903** [comment]
:+1:

**Comment by foutoucour** [comment]
:+1:

**Comment by foutoucour** [comment]
:+1:

**Comment by digitalsatori** [comment]
:100:

**Pull request merged**